src/chrome/content/rules/AJC.com.xml

```diff
@@ -1,27 +1,17 @@
 <!--
-	^ajc.com: Handshake fails
-	www.ajc.com: 504, Akamai
-
-
-	Problematic hosts in *ajc.com:
-
-		- *.blog *
-
-	* Mismatched
+	Other Atlanta Journal Constitution rulesets:
+		* MyAJC.com.xml
 
+	Numerous unsupported subdomains (>300 found by Sublist3r)
+	Mixed content on some www.ajc.com pages
 -->
-<ruleset name="AJC.com (partial)" default_off="mismatched">
-
-	<!--	Direct rewrites:
-				-->
-	<target host="*.blog.ajc.com" />
-
-		<test url="http://music.blog.ajc.com/" />
-		<test url="http://newstome.blog.ajc.com/" />
-
-
-	<rule from="^http:"
-		to="https:" />
-
+<ruleset name="AJC.com (partial)">
+	<target host="ajc.com" />
+	<target host="www.ajc.com" />
+	<target host="myaccount.ajc.com" />
+	<target host="store.ajc.com" />
+	<target host="subscribe.ajc.com" />
+
+	<rule from="^https?://ajc\.com/" to="https://www.ajc.com/"/>
```

Comment:
I want you to put a comment in this ruleset to explain why touching https is required in this case.
Maybe you can also point to some example where we can reproduce the issue.